src/pages/Gender.js

```diff
@@ -0,0 +1,7 @@
+export default function Age() {
```

Comment:
Check the function name, it's called `Age`. Make sure you call it Gender as well